xdebug.c

```diff
@@ -81,9 +81,9 @@ static int xdebug_post_startup(void);
 #endif
 
 /* error callback replacement functions */
-void (*xdebug_old_error_cb)(int type, const char *error_filename, const uint error_lineno, const char *format, va_list args) ZEND_ATTRIBUTE_PTR_FORMAT(printf, 4, 0);
-void (*xdebug_new_error_cb)(int type, const char *error_filename, const uint error_lineno, const char *format, va_list args);
-void xdebug_error_cb(int type, const char *error_filename, const uint error_lineno, const char *format, va_list args);
+void (*xdebug_old_error_cb)(int type, const char *error_filename, const unsigned int error_lineno, const char *format, va_list args) ZEND_ATTRIBUTE_PTR_FORMAT(printf, 4, 0);
```

Comment:
uint32_t for all lineno args.

Reply:
Also true for PHP 7.0- 7.3?

Reply:
Thanks! Done.

Reply:
Nope:

Reply:
It's okay, as sizeof(unsigned int) == sizeof(uint32_t) on all platforms you're likely to target. It should just use the correct type going forward.